apps/stable_diffusion/web/utils/gradio_configs.py

```diff
@@ -0,0 +1,31 @@
+import os
+import tempfile
+import gradio
+from os import listdir
+
+gradio_tmp_imgs_folder = os.getcwd() + "/shark_tmp/"
+
```

Comment:
Please fix lint error and also avoid using such additions. Use os.path.join() or Path() for concatenating directories.

Reply:
Hello, and thank for the feedback.
I'm not really sure for "shark_tmp/", I noticed there is also a os.sep, on the other hand, I didn't invented this path and found a "./shark_tmp/" in the existing code, so I supposed it's ok.
Sorry about that, I'm not very experienced in Python.